api/src/main/java/com/capitalone/dashboard/service/DashboardRemoteServiceImpl.java

```diff
@@ -56,18 +59,51 @@ public DashboardRemoteServiceImpl(
         this.componentRepository = componentRepository;
     }
 
+    /**
+     * Creates a list of unique owners from the owner and owners requests
+     * @param request
+     * @return List<Owner> list of owners to be added to the dashboard
+     * @throws HygieiaException
+     */
+    private List<Owner> getOwners(DashboardRemoteRequest request) throws HygieiaException {
+        DashboardRemoteRequest.DashboardMetaData metaData = request.getMetaData();
+        Owner owner = metaData.getOwner();
+        List<Owner> owners = metaData.getOwners();
+
+        if (owner == null && owners == null) {
+            throw new HygieiaException("There are no owner/owners field in the request", HygieiaException.INVALID_CONFIGURATION);
+        }
+
+        if (owners == null) {
+            owners = new ArrayList<Owner>();
+            owners.add(owner);
+        } else if (owner != null) {
```

Comment:
This condition is unnecessary

Reply:
Since we don't denote @NotNull for DashboardMetaData, don't we have to check here if one is null?  If someone only passes in owner, the owners field will be null, and likewise for passing in owners, the owner field will be null.  I tried removing this condition but it fails with "internal error" if only one of the fields is given.